src/app/beer_garden/events/processors.py

```diff
@@ -20,10 +20,12 @@ def __init__(self, action=None, **kwargs):
         super().__init__(**kwargs)
 
         self._action = action
+        self._processing = True
```

Comment:
This shouldn't be in the base class

Reply:
Done, moved to the `HttpParentUpdater` class only